smartlearner/utils.py

```diff
@@ -23,3 +25,44 @@ def save_dict_to_json_file(path, dictionary):
 def load_dict_from_json_file(path):
     with open(path, "r") as json_file:
         return json.loads(json_file.read())
+
+
+def split_dataset(dataset, proportions):
```

Comment:
`proportions` should be a list of values between 0 and 1 summing to one. If not, you should make it clear what it is in the docstring.